contracts/safe-hts-precompile/SafeOperations.sol

```diff
@@ -6,6 +6,9 @@ import "./SafeHTS.sol";
 
 contract SafeOperations {
 
+    event TokenCreated(address tokenAddress);
+    event TokenInfoEvent(address token, IHederaTokenService.TokenInfo tokenInfo);
```

Comment:
nit: both address params should probably have the same name, otherwise it kind of implies that `token` is somehow different from `tokenAddress`